src/pages/SetPasswordPage.js

```diff
@@ -132,4 +137,4 @@ SetPasswordPage.defaultProps = defaultProps;
 export default withOnyx({
     credentials: {key: ONYXKEYS.CREDENTIALS},
     account: {key: ONYXKEYS.ACCOUNT},
-})(SetPasswordPage);
+})(withWindowDimensions((SetPasswordPage)));
```

Comment:
We use a `compose()` method to combine` withOnyx()` and `withWindowDimensions()`